kube/services/k8resources/gro/gro-rc.yaml

```diff
@@ -42,14 +42,15 @@ spec:
               add_header X-Frame-Options "SAMEORIGIN" always;
               add_header X-Content-Type-Options "nosniff" always;
               add_header X-XSS-Protection "1; mode=block" always;
+              proxy_cache on;
```

Comment:
Based on the nginx docs this should be _enough_ to enable caching from the app for hopefully just this location.  It's slightly unclear enough, as you set a zone but I haven't set proxy_cache_path which defines the "zone".  Apparently it defaults though, but to what? O.o